source/commands/env/template/apply.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { option } from 'pastel';
+import zod from 'zod';
+import { type infer as zInfer } from 'zod';
+import { AuthProvider } from '../../../components/AuthProvider.js';
+import ApplyComponent from '../../../components/env/template/ApplyComponent.js';
+
+export const description = 'A apply command to run the TF file';
+
+export const options = zod.object({
+	key: zod
+		.string()
+		.optional()
+		.describe(
+			option({
+				description:
+					'Optional: API Key to be used for the environemnt to apply the terraform template.',
```

Comment:
```suggestion
					'Optional: API Key to be used for the environemnt to apply the policy template.',
```

Please do not use terraform explicitly and refer it as `environment template` or `policy template`